jobs.groovy.j2

```diff
@@ -442,6 +442,9 @@ branches.each {
     wrappers {
       timestamps()
       colorizeOutput()
+      timeout {
+          absolute(45)
+          }
     }
```

Comment:
but lock down the multijob to 45 minutes so that 'stuck' jobs won't block future runs